payroll/services.py

```diff
@@ -129,4 +131,12 @@ def _get_bills_queryset(self, obj_data):
             status__in=[Bill.Status.VALIDATED]
         )
 
+        if included_unpaid:
+            bills_queryset = bills_queryset.filter(json_ext__unpaid=True)
+        else:
+            bills_queryset = bills_queryset.filter(
+                Q(json_ext__unpaid=False) |
+                Q(json_ext__unpaid__isnull=True)
+            )
+
```

Comment:
Have you tested this on MSSQL db? I don't think it'll work. Since we have new CI can you rebase this branch do develop and write some unit test for this function to confirm if it'll work on both DB engines?

Reply:
Yes, it works also on MSSQL engine. Nevertheless I will add simple unit test to cover this part of code to ensure it works on both db engines.